### PR TITLE
Skip over partial methods with no implementation when emitting extra …

### PR DESCRIPTION
**Pull request by VSadov**
…PDB info for winmdobj

### CODE REVIEW

**Comment by VSadov** [comment]
@AlekseyTs @gafter @jaredpar @khyperia - please review.

**Comment by AlekseyTs** [comment]
LGTM

File: src/Compilers/CSharp/Portable/Emitter/Model/PEModuleBuilder.cs [hunk 1]
```diff
@@ -242,10 +242,13 @@ internal sealed override IEnumerable<Cci.INestedTypeDefinition> GetSynthesizedNe
                                         // NOTE: Dev11 does not add synthesized static constructors to this map,
                                         //       but adds synthesized instance constructors, Roslyn adds both
                                         var method = (MethodSymbol)member;
-                                        if (!method.IsDefaultValueTypeConstructor())
+                                        if (method.IsDefaultValueTypeConstructor() ||
+                                            method.IsPartialMethod() && (object)method.PartialImplementationPart == null)
```
Comment (gafter) [inline src/Compilers/CSharp/Portable/Emitter/Model/PEModuleBuilder.cs]: This should be `method.CallsAreOmitted(null)`.

**Comment by jaredpar** [comment]
LGTM

**Comment by gafter** [comment]
LGTM

**Comment by MattGertz** [comment]
Approved 5/22 by ML Shiproom.

File: src/Compilers/CSharp/Portable/Emitter/Model/PEModuleBuilder.cs (hunk 1 above)
Comment (AlekseyTs) [inline src/Compilers/CSharp/Portable/Emitter/Model/PEModuleBuilder.cs]: Current condition matches condition in GetMethodsToEmit. If we decide to change it, both places should be changed.

**Pull request merged**